webapp/components/abstract-test-file-results-table.html

```diff
@@ -8,19 +8,74 @@
   <script>
```

Comment:
Broad point of clarification: Aren't these tables usually embedded in the context of other components that perform fetch? Are we sure that this addition isn't double-fetching for some view configurations?

Reply:
Thanks for spotting this! Although my refactor might be a little bit cleaner logically, we don't want double fetching. I just reverted my refactor. Let's fix the bug first.